docs/examples/floating-pv/plot_floating_pv_cell_temperature.py

```diff
@@ -0,0 +1,241 @@
+r"""
+Calculating the cell temperature for floating PV
+================================================
+
+This example uses the PVSyst temperature model to calculate the
+cell temperature for floating photovoltaic (FPV) systems.
+
+One of the primary benefits attributed to FPV systems
+is lower operating temperatures, which are expected to increase the
+operating efficiency. In general, the temperature at which a photovoltaic
+module operates is influenced by various factors including solar radiation,
+ambient temperature, wind speed and direction, and the characteristics of the
+cell and module materials, as well as the mounting structure. Both radiative
+and convective heat transfers play roles in determining the module's
+temperature.
+
+A popular model for calculating the PV cell temperature is the
+empirical heat loss factor model suggested by Faiman. A modified version of
+this model is implemented in PVSyst
+(:py:func:`~pvlib.temperature.pvsyst_cell`). The PVSyst model for cell
+temperature :math:`T_{C}` is given by
+
+.. math::
+    :label: pvsyst
+
+    T_{C} = T_{a} + \frac{\alpha \cdot E \cdot (1 - \eta_{m})}{U_{c} + U_{v} \cdot WS}
+
+Where :math:`E` is the plane-of-array irradiance, :math:`T_{a}` is the
```

Comment:
I always punctuate like this, but I know not everyone likes it. Just a suggestion

```suggestion
temperature :math:`T_{C}` is given by:

.. math::
    :label: pvsyst

    T_{C} = T_{a} + \frac{\alpha \cdot E \cdot (1 - \eta_{m})}{U_{c} + U_{v} \cdot WS},

where :math:`E` is the plane-of-array irradiance, :math:`T_{a}` is the
```